modules/layers/src/arc-layer/arc-layer-uniforms.ts

```diff
@@ -0,0 +1,39 @@
+import {ShaderModule} from '@luma.gl/shadertools';
+import {UniformTypes} from '@deck.gl/core';
+
+const uniformBlock = `\
```

Comment:
Thought:

There is unused code in luma.gl that automatically generates these  shader uniform blocks from the module definitions (both for GLSL and WGSL). If you continue to break these out we may be able to activate that code at some point. 

Depends on whether we feel comfortable enough with auto-generated shader code as it can get a bit more subtle when debugging.

Reply:
Agree, that might be nice in the future - but I would prefer to be explicit until we have full WebGL2&WebGPU support. It's hard enough debugging this code as is :)